*** ./smali/android/nfc/NfcManager.smali	2014-04-22 17:18:38.597462826 +0800
--- /home/xinxin520/patchrom/miuiv5/u8500/temp/new_smali/framework.jar.out/./smali/android/nfc/NfcManager.smali	2014-04-22 17:16:54.805463390 +0800
***************
*** 64,66 ****
--- 64,79 ----
  
      return-object v0
  .end method
+ 
+ .method public setPollDuration(I)I
+     .locals 1
+     .parameter "pollDurationMs"
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->NEW_METHOD:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
+ 
+     .prologue
+     const/4 v0, 0x0
+ 
+     return v0
+ .end method
