*** ./smali/android/preference/PreferenceGroup.smali	2014-04-01 01:04:07.274475077 +0800
--- /home/xiangxin/miuiv5/u8500/temp/new_smali/framework.jar.out/./smali/android/preference/PreferenceGroup.smali	2014-04-01 01:02:22.790475645 +0800
***************
*** 257,262 ****
--- 257,264 ----
      :cond_5
      invoke-virtual {p0}, Landroid/preference/PreferenceGroup;->notifyHierarchyChanged()V
  
+     invoke-virtual {p1, p0}, Landroid/preference/Preference;->setParent(Landroid/preference/PreferenceGroup;)V
+ 
      move v1, v2
  
      goto :goto_0
