*** ./smali/android/telephony/SmsMessage.smali	2014-04-01 01:04:59.258474794 +0800
--- /home/xiangxin/miuiv5/u8500/temp/new_smali/framework.jar.out/./smali/android/telephony/SmsMessage.smali	2014-04-01 01:03:14.918475360 +0800
***************
*** 825,830 ****
--- 825,839 ----
      return-object v0
  .end method
  
+ .method public getMessageSimId()I
+     .locals 1
+ 
+     .prologue
+     const/4 v0, 0x0
+ 
+     return v0
+ .end method
+ 
  .method public getOriginatingAddress()Ljava/lang/String;
      .locals 1
  
