*** ./smali/com/android/internal/os/ZygoteInit.smali	2014-04-22 17:18:47.033462780 +0800
--- /home/xinxin520/patchrom/miuiv5/u8500/temp/new_smali/framework.jar.out/./smali/com/android/internal/os/ZygoteInit.smali	2014-04-22 17:17:02.789463347 +0800
***************
*** 419,424 ****
--- 419,426 ----
      :try_start_0
      invoke-static {}, Lcom/android/internal/os/SamplingProfilerIntegration;->start()V
  
+     invoke-static {}, Lmiui/security/SecurityManager;->init()V
+ 
      invoke-static {}, Lcom/android/internal/os/ZygoteInit;->registerZygoteSocket()V
  
      const/16 v2, 0xbcc
