*** ./smali/com/android/internal/os/ZygoteInit.smali	2014-04-01 01:05:07.034474752 +0800
--- /home/xiangxin/miuiv5/u8500/temp/new_smali/framework.jar.out/./smali/com/android/internal/os/ZygoteInit.smali	2014-04-01 01:03:21.598475324 +0800
***************
*** 419,424 ****
--- 419,426 ----
      :try_start_0
      invoke-static {}, Lcom/android/internal/os/SamplingProfilerIntegration;->start()V
  
+     invoke-static {}, Lmiui/security/SecurityManager;->init()V
+ 
      invoke-static {}, Lcom/android/internal/os/ZygoteInit;->registerZygoteSocket()V
  
      const/16 v2, 0xbcc
